cicecore/cicedyn/infrastructure/ice_read_write.F90

```diff
@@ -1241,7 +1241,7 @@ subroutine ice_read_nc_xy(fid,  nrec,  varname, work,  diag, &
          status = nf90_get_att(fid, varid, "_FillValue", missingvalue)
 !          call ice_check_nc(status, subname//' ERROR: Missing _FillValue', &
 !                            file=__FILE__, line=__LINE__)
-         write(nu_diag,*) subname,' missingvalue= ',missingvalue
+!         write(nu_diag,*) subname,' missingvalue= ',missingvalue
```

Comment:
I did not read the full discussion, but is there any reason why we are commenting these lines instead of removing them ?

Reply:
There a bit of useful debugging code that is part of the optional file read diagnostics that is turned off by default.  It's sort of helpful to have this code around in case it's needed.